devtools/stdio-sse-proxy/src/main/java/io/quarkiverse/mcp/server/proxy/StdioSseProxy.java

```diff
@@ -1,7 +1,7 @@
 //usr/bin/env jbang "$0" "$@" ; exit $?
 //DEPS io.quarkus.platform:quarkus-bom:3.18.3@pom
 //DEPS io.quarkus:quarkus-picocli
-//Q:CONFIG quarkus.log.console.stderr=true
+//FILES application.properties=../../../../../../resources/application.properties
```

Comment:
Why this change? I don't like the `../../../../../../resources/application.properties` part.

Reply:
so it uses the same application.properties as the mvn build.

but ultimately if just getting this published to maven central it does not need to be a jbang script so can remove it all

Reply:
Hm, `quarkus.log.console.stderr=true` is the only property that matters here and I'd like to keep the possibility to run this as a jbang script.

Reply:
> sure - but above keeps letting you run it as a jbang script.

Yes, but that's the intention. You can run the deployed artifact or the latest version as a script.

> anyhow - no big deal; I can remove it if you want/need?

+1

Reply:
ok so just tested this and this need to use //FILES to work because https://github.com/quarkusio/quarkus/issues/44466 is still an issue.

runtime/buildtime config does not get honored since the config system changed.

Reply:
Funny, it worked just fine when I tested the PR but maybe I missed something... :thinking:

Reply:
If I understand the description of https://github.com/quarkusio/quarkus/issues/44466 correctly then `//Q:CONFIG` works but the solution is suboptimal and should be refactored.

@maxandersen I will remove this change and update the readme.

Reply:
well it means that //Q:CONFIG for many cases will only take effect if jbang does a build - not when just running jar directly. Basically - //Q:CONFIG is not useful unless doing development.